gnd/src/main/res/layout/feature_sheet_header.xml

```diff
@@ -15,72 +15,86 @@
+            android:layout_height="wrap_content"
+            android:text="@{viewModel.featureTitle}"
+            android:textAlignment="center"
+            android:textAppearance="@style/TextAppearance.AppCompat.Title"
+            tools:text="Feature Sheet Title" />
+
+        <TextView
+            android:id="@+id/feature_sheet_subtitle"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@{viewModel.featureSubtitle}"
+            android:textAlignment="center"
+            android:textColor="@color/colorGrey600"
+            android:textSize="12sp"
+            android:visibility="@{viewModel.featureSubtitleVisibility}"
+            tools:text="Feature Subtitle" />
+      </LinearLayout>
+    </androidx.cardview.widget.CardView>
+
+    <!-- HACK: Scrim to obscure bottom rounded corners -->
+    <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/feature_sheet_corner_radius"
+        android:layout_gravity="bottom"
+        android:background="@color/colorBackground"
+        android:elevation="3dp" />
 
-  <!-- TODO: Add drop shadow to top of circle and card -->
-  <ImageView
-    android:id="@+id/feature_header_icon_background"
-    android:layout_width="@dimen/feature_header_icon_background_diameter"
-    android:layout_height="@dimen/feature_header_icon_background_diameter"
-    android:layout_gravity="top|center_horizontal"
-    android:src="@drawable/feature_header_icon_background_drawable"
-    android:elevation="4dp"/>
+    <!-- TODO: Add drop shadow to top of circle and card -->
+    <ImageView
+        android:id="@+id/feature_header_icon_background"
+        android:layout_width="@dimen/feature_header_icon_background_diameter"
+        android:layout_height="@dimen/feature_header_icon_background_diameter"
+        android:layout_gravity="top|center_horizontal"
+        android:elevation="4dp"
+        android:src="@drawable/feature_header_icon_background_drawable" />
 
-  <ImageView
-    android:id="@+id/feature_header_icon"
-    android:paddingTop="@dimen/feature_header_icon_top_padding"
-    android:layout_width="@dimen/feature_header_icon_width"
-    android:layout_height="@dimen/feature_header_icon_height"
-    android:layout_gravity="top|center_horizontal"
-    app:srcCompat="@drawable/ic_default_map_marker"
-    android:tint="@color/colorForeground"
-    android:elevation="5dp"/>
+    <ImageView
+        android:id="@+id/feature_header_icon"
+        android:layout_width="@dimen/feature_header_icon_width"
+        android:layout_height="@dimen/feature_header_icon_height"
+        android:layout_gravity="top|center_horizontal"
+        android:elevation="5dp"
+        android:paddingTop="@dimen/feature_header_icon_top_padding"
+        android:tint="@color/colorForeground"
+        app:srcCompat="@drawable/ic_default_map_marker" />
 
-</FrameLayout>
+  </FrameLayout>
+</layout>
```

Comment:
Please only reformat existing unchanged code in a separate PR so that diffs are more meaningful.

Reply:
Would it be helpful if I create multiple commits clearly specifying which ones only contain auto-formatting so it's one less commit to review. Or would you still prefer separate PRs for that?

Reply:
Actually since it's possible can review each commit separately for this PR it's ok to just reformat in this PR. In future PRs we can separate reformatting from logic changes.